ext/build_docs.py

```
@@ -0,0 +1,77 @@
+
+import json
+import os
+import subprocess
+import sys
+
+KATEX_HEADER = "gh-pages/katex-header.html"
+EXCLUDED_FEATURES = {"default", "gpu"}
+
+
+def metadata(*args):
+    out = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)